cwms-servers/CWMS_HOME/config/system_config/cwms_system_logging.properties

```diff
@@ -0,0 +1,61 @@
+#
+######################################################
+#                                                    #
+#  This file should only be edited by HEC personnel  #
+#  Or, only at their direction                       #
+#                                                    #
+######################################################
+#
+#
+#  Specify the handlers to create in the root logger
+#  (all loggers are children of the root logger)
+#  Create handlers
+
+# handlers = java.util.logging.ConsoleHandler,com.splunk.logging.HttpEventCollectorLoggingHandler,biz.paluch.logging.gelf.jul.GelfLogHandler
+handlers = java.util.logging.ConsoleHandler,biz.paluch.logging.gelf.jul.GelfLogHandler
+
+#  Set the default logging level for the root logger
+
+.level = INFO
+
+#   Set the default logging level for new ConsoleHandler instances
+# java.util.logging.ConsoleHandler.level = INFO
+
+java.util.logging.ConsoleHandler.level = ALL
+
+# Set the default formatter for new ConsoleHandler instances
+# java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
+
+java.util.logging.ConsoleHandler.formatter = rma.util.logging.CustomFormatter
+
+#      Format elements
+# * <li>{@code %m} - message</li>
+# * <li>{@code %L} - log level</li>
+# * <li>{@code %n} - name of the logger</li>
+# * <li>{@code %t} - a timestamp (in ISO-8601 "yyyy-MM-dd HH:mm:ss.SSS" format)</li>
+# * <li>{@code %M} - source method name (if available, otherwise "?")</li>
+# * <li>{@code %c} - source class name (if available, otherwise "?")</li>
+# * <li>{@code %C} - source simple class name (if available, otherwise "?")</li>
+# * <li>{@code %T} - thread ID</li>
+# * <li>{@code %E} - (Filename.java:linenumber) Slow to generate Eclipse format</li>
+
+#     Sample log formats
+# rma.util.logging.CustomFormatter.format = %L:>>  %m
+# rma.util.logging.CustomFormatter.format = %t %L: %C.%M %m
+# rma.util.logging.CustomFormatter.date.format = ddMMMyyyy HH:mm:ss
+# rma.util.logging.CustomFormatter.date.timezone = UTC
+
+rma.util.logging.CustomFormatter.format = %t %L: %m
+rma.util.logging.CustomFormatter.date.format = HH:mm:ss
+
+#Logstash Log Handler Setup
+biz.paluch.logging.gelf.jul.GelfLogHandler.host = http:<host>
+biz.paluch.logging.gelf.jul.GelfLogHandler.port = <port>
+biz.paluch.logging.gelf.jul.GelfLogHandler.facility = "NWP CWMS v3.3"
```

Comment:
What does this correspond to?

Reply:
well, the documentation no longer says since it's deprecated. Removed